kotlin-checks-test-sources/src/main/kotlin/androidx/compose/material/OutlinedTextField.kt

```diff
@@ -0,0 +1,30 @@
+package androidx.compose.material
+
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.foundation.text.input.TextFieldState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.VisualTransformation
+
+// Partial stubbing of parameters
```

Comment:
I added these comments about partial stubbing so that it will be easier to recognize for which methods the stub has missing parameters, and add if needed.
I couldn't have complete stubs with all parameters because many functions and constructors take a large number of parameters, that would have forced me to mock way more classes.